src/components/SendTransaction/CreateTransaction.tsx

```diff
@@ -153,12 +188,121 @@ export const CreateTransaction = ({
     onInput({ ...formData, federationAddress: undefined });
   };
 
+  const validateInput = (inputId: string) => {
+    const errors: ValidatedInput = {};
+    let message = "";
+    const memoContent = formData.memoContent as string;
+
+    switch (inputId) {
+      case SendFormIds.SEND_TO:
+        if (!formData.toAccountId) {
+          message = "Please enter a valid Stellar or Federated address";
+        } else if (
+          !isFederationAddress(formData.toAccountId) &&
+          !StrKey.isValidEd25519PublicKey(formData.toAccountId)
+        ) {
+          message = "Stellar address or public key is invalid";
+        }
+
+        errors[SendFormIds.SEND_TO] = message;
+        break;
+      case SendFormIds.SEND_AMOUNT:
+        if (formData.amount.lte(0)) {
+          message = "Amount must be larger than 0";
+        } else if (formData.amount.gt(availableBalance)) {
+          message = "This amount is larger than your balance";
+        }
+
+        errors[SendFormIds.SEND_AMOUNT] = message;
+        break;
+      case SendFormIds.SEND_FEE:
+        // recommendedFee is minimum fee
+        errors[SendFormIds.SEND_FEE] = new BigNumber(maxFee).lt(recommendedFee)
```

Comment:
I think it's okay if we allow them to set their maxFee lower than the recommended fee right? Even though their transaction is more likely to fail, that's their decision to risk that.

Reply:
Current AV doesn't do that, I'll check with Product if we want to change it. If we do, I'll create another PR for it.